docs/api/en/core/Object3D.html

```diff
@@ -35,7 +35,7 @@ <h3>[property:AnimationClip animations]</h3>
 		<h3>[property:Boolean castShadow]</h3>
 		<p>Whether the object gets rendered into shadow map. Default is *false*.</p>
 
-		<h3>[property:Object3D children]</h3>
+		<h3>[property:Array children]</h3>
```

Comment:
At some point, we should document such properties or return types as: `[property:Array<Object3D> children]`.

Unfortunately, the docs do no support such a markup yet. Maybe we can add something like this in a future PR?